unicoremc/models.py

```diff
@@ -498,3 +539,16 @@ def destroy(self):
                 self.app_type, self.country)
 
         self.db_manager.destroy(self.app_type, self.country)
+
+
+@receiver(post_save, sender=Project)
+def publish_to_websocket(sender, instance, created, **kwargs):
```

Comment:
@smn probably the most interesting thing here - post_save signal to broadcast when a project has been saved.